### ISSUE TITLE
Training accuracy problem

### ISSUE THREAD

**Issue by wycm2022** [tsingqguo/efficientderain#4]
Hello, thank you for releasing the code. 

But according to the train.sh you provided, the accuracy of PSNR obtained by training the rain100H dataset is only about 23~24, and the SSIM is about 0.77. I don’t know what went wrong. 

Can you give me some advice? Thanks again for your work

**Comment by MKFxxL** [comment]
1. I'm sorry that I forgot to add the rainlayers for rainmix, now I added them into the rainmix folder. Maybe you need to redownload them.
2. I uploaded the pretrained models for SPA-data, you can test using these models.
3. Here are some tips which may help: 
  (1) For dataset rain1400, we trained 200~240 epochs to get the best model of v2. And we fine-tune on the pretrained model of v2 for about 40 epochs to get the best model of v3, and about 60 epochs to get the best model for v4. (We used smaller learning rate for fine-tuning)
  (2) For dataset rain100H, we trained 3500~4000 epochs to get the best model of v2. And we fine-tune on the pretrained model of v2 for about 2500 epochs to get the best model of v3, and about 2400 epochs to get the best model for v4. (We used smaller learning rate for fine-tuning)
  (3) For dataset rain100L, we trained about 4800 epochs to get the best model of v3, and we trained about 600 epochs to get the best model of v4.
  (4) For dataset SPA-data, we trained about 351k iters to get the best model of v3, and we trained about 117k iters to get the best model of v4. (batch size is 16)

**Comment by MKFxxL** [comment]
> Hello, thank you for releasing the code.
> 
> But according to the train.sh you provided, the accuracy of PSNR obtained by training the rain100H dataset is only about 23~24, and the SSIM is about 0.77. I don’t know what went wrong.
> 
> Can you give me some advice? Thanks again for your work

1. I'm sorry that I forgot to add the rainlayers for rainmix, now I added them into the rainmix folder. Maybe you need to redownload them.
2. I uploaded the pretrained models for SPA-data, you can test using these models.
3. Here are some tips which may help: 
    (1) For dataset rain1400, we trained 200 to 240 epochs to get the best model of v2. And we fine-tune on the pretrained model of v2 for about 40 epochs to get the best model of v3, and about 60 epochs to get the best model for v4. (We used smaller learning rate for fine-tuning) 
    (2) For dataset rain100H, we trained 3500 to 4000 epochs to get the best model of v2. And we fine-tune on the pretrained model of v2 for about 2500 epochs to get the best model of v3, and about 2400 epochs to get the best model for v4. (We used smaller learning rate for fine-tuning)
     (3) For dataset rain100L, we trained about 4800 epochs to get the best model of v3, and we fine-tune on the pretrained model of v3 for about 600 epochs to get the best model of v4.
     (4) For dataset SPA-data, we trained about 351k iters to get the best model of v3, and we fine-tune on the pretrained model of v3 for about 117k iters to get the best model of v4. (batch size is 16)
